src/routes/dashboard.tsx

```diff
@@ -30,7 +30,7 @@ export default function Dashboard() {
     const results = useQueries({
         queries: config.sections.flatMap(section => {
             return config.connections.map((connection, idx) => ({
-                queryKey: ['pulls', connection.host, section.search],
+                queryKey: ['pulls', connection.host, connection.auth, section.search],
```

Comment:
I was considering as an alternative using the connection name that you introduce in #12. But at the same time using the token is a better representation of the identity of the token (e.g., if we switch the names of two connections we should not reuse the cache of the other one).